mac-insurance-frontend/components/TabSelector.js

```diff
@@ -5,17 +5,21 @@ import { Pool } from "./Pool";
 import { TabButtonContainer } from "./TabButtonContainer";
 import { WalletDashboard } from "./WalletDashboard";
 
-export const TabSelector = ({ walletConnected }) => {
+export const TabSelector = ({ walletConnected, account }) => {
   const [selected, setSelected] = useState("Dashboard");
 
   const renderTab = () => {
     switch (selected) {
       case "Markets":
-        return <Markets />;
+        return <Markets account={account} />;
       case "Pool":
         return <Pool />;
       default:
-        return !walletConnected ? <NoWalletDashboard /> : <WalletDashboard />;
+        return !walletConnected ? (
+          <NoWalletDashboard />
+        ) : (
+          <WalletDashboard account={account} />
```

Comment:
I added the account in the wallet dashboard as well so that we can retrieve list of accounts